Samples/NetworkMediaStreamer/NetworkMediaStreamerBase/inc/pch.h

```diff
@@ -0,0 +1,17 @@
+// Copyright (C) Microsoft Corporation. All rights reserved.
+#pragma once

```

Comment:
For public sample code, the copyright should be this, per [https://microsoft.sharepoint.com/sites/celaweb-copyrights-trademarks-and-patents/sitepages/copyrights-legal-notices.aspx](https://microsoft.sharepoint.com/sites/celaweb-copyrights-trademarks-and-patents/sitepages/copyrights-legal-notices.aspx)

Please cross-check my reading of that page...

//    Copyright (c) Microsoft Corporation. All rights reserved.
//    This code is licensed under the Microsoft Public License.
//    THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
//    ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
//    IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
//    PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.

Reply:
The one you have pasted is MS public license
i thought we are doing MIT license for these samples.
[http://vschart.com/compare/microsoft-public-license/vs/gnu-affero-general-public-license](http://vschart.com/compare/microsoft-public-license/vs/gnu-affero-general-public-license)


---
In reply to: [491097076](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491097076) [](ancestors = 491097076)

Reply:
Then put in MIT license. The one liner you have right now isn't MIT license - or is it sufficient?

---
In reply to: [491219434](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491219434) [](ancestors = 491219434,491097076)

Reply:
I saw some other microsoft projects under MIT license use the following -
 
/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------*/
I shall use this unless someone has concerns. Please let me know

Reply:
I saw some other microsoft projects under MIT license use the following -

/---------------------------------------------------------------------------------------------
 *  Copyright (c) Microsoft Corporation. All rights reserved.
 *  Licensed under the MIT License. See License.txt in the project root for license information.
 *--------------------------------------------------------------------------------------------/
I shall use this unless someone has concerns. Please let me know 

---
In reply to: [491242437](https://github.com/microsoft/Windows-Camera/pull/13#discussion_r491242437) [](ancestors = 491242437,491219434,491097076)